contrib/test.bzl

```diff
@@ -160,13 +160,14 @@ def container_test(name, image, configs, driver = None, verbose = None, **kwargs
         # Include the package name in the new image tag to avoid conflicts on naming
         # when running multiple container_test on images with the same target name
         # from different packages.
-        sanitized_name = (native.package_name() + image).replace(":", "").replace("@", "").replace("/", "")
+        sanitized_name = (native.package_name() + image).replace(":", "").replace("@", "").replace("/", "").lower()
```

Comment:
why the need to lowe case this?

Reply:
Ah, because it gets used in a container's name i believe and uppercase characters are not allowed there.  So if you have a `container_test(name = "badPractices")`, it wouldn't work as-is.